### PR TITLE
Add CI to auto update example PDFs.

### PR DESCRIPTION
**Pull request by yjyao**
Also add pointer to example PDFs in README.

### CODE REVIEW

**Comment by passamo9** [comment]
Everything okay here?

image

**Comment by yjyao** [comment]
> Everything okay here?

lol apparently not. i made a typo in the config. working on fix.

**Comment by yjyao** [comment]
passing.

**Comment by passamo9** [comment]
@yjyao What do you need for me to review?

**Comment by bond7o6** [comment]
This intended?

image

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,3 @@
 *.log
 *.gz
 *.pdf
```
Comment (Giebisch) [inline .gitignore:6]:
```suggestion
```
Don't you want to add a pdf ?

File: .github/workflows/latexmk.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            pleading.tex
+          working_directory:
+            examples
+          latexmk_use_xelatex: true
+      - uses: fregante/setup-git-user@v1
+      - name: Upload PDF to examples branch
+        run: |
+          git fetch origin examples
+          git checkout -b examples origin/examples --merge
+          git add *.pdf
```
Comment (Giebisch) [inline .github/workflows/latexmk.yml:22]:
```suggestion
          git add *
```
Add all, other files will be ignored anyway.

File: .gitignore (hunk 1 above)
Comment (yjyao) [reply .gitignore:6]: the example branch is an empty branch with no `.gitignore`. this `.gitignore` only affects developers.

Comment (Giebisch) [reply .gitignore:6]: Ah got it, nevermind my comment then. As you see I'm not too familiar with Github Actions ;)

**Comment by yjyao** [comment]
@bond7o6 fixed CI. sorry i don't have a good way of testing the CIs so i had to commit and see what happens.

> @passamo9: What do you need for me to review?

- check that the CIs/actions are passing like you did in [your first comment](https://github.com/passamo9/LaTeX_EnglandWales_Templates/pull/18#issuecomment-778847483)
- check the changed README.md looks good to you.

**Comment by passamo9** [comment]
- CI seems good 
image

-  README.md seems to have a problem: 
image

**Comment by yjyao** [comment]
@passamo9 fixed README. feel free to merge.

**Review by Giebisch — APPROVED** [review]

**Pull request merged**